content/id/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Controllers
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Dalam bidang robotika dan otomatisasi, _control loop_ adalah
+lingkaran tertutup yang mengatur keadaan suatu sistem.
+
+Berikut adalah salah satu contoh loop kontrol: termostat di sebuah ruangan.
+
+Ketika Anda mengatur suhunya, itu mengisyaratkan ke termostat
+tentang *keadaan yang Anda inginkan*. Sedangkan suhu kamar yang sebenarnya 
+adalah *keadaan saat ini*. Thermostat berfungsi untuk membawa keadaan saat ini
+mendekati ke keadaan yang diinginkan, dengan menghidupkan atau mematikan 
+perangkat.
+
+Di Kubernetes, controller adalah kontrol tertutup yang mengawasi keadaan cluster
+{{< glossary_tooltip term_id="cluster" >}} Anda, lalu membuat atau meminta 
+perubahan jika diperlukan. Setiap controller mencoba untuk memindahkan status 
+cluster saat ini mendekati keadaan yang diinginkan.
+
+{{< glossary_definition term_id="controller" length="short">}}
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Pola controller 
+
+Sebuah controller melacak sekurang-kurangnya satu jenis *resources* dari 
+Kubernetes.
+[objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/) ini
+memiliki *spec field* yang merepresentasikan keadaan yang diinginkan. Satu atau 
+lebih controller untuk *resource* tersebut bertanggung jawab untuk membuat 
+keadaan sekarang mendekati keadaan yang diinginkan.
+
+Controller mungkin saja melakukan tindakan itu sendiri; namun secara umum, di 
```

Comment:
Need to italicize all "controller" in this page because it's imported word
```suggestion
_Controller_ mungkin saja melakukan tindakan itu sendiri; namun secara umum, di 
```